### ISSUE TITLE
Missing STUDY_ID and PATID linkage in PCORNET_TRIAL table - UU

### ISSUE THREAD

**Issue by sxinger** [gpcnetwork/r61-r33_vccc_kumc#7]
There are 8 participants from UU that we are still missing the linkage between the STUDY_ID and PCORnet PATID

**Comment by Jackson-Barlocker** [comment]
Updating the GitHub ticket for visibility.

I have been working with Song offline to discuss the missing 8 patients. 3 of the patients are ready to be sent, the other 5 have a caveat of being deemed a "VIP" in our system. Generally, VIPs are not allowed to have their data used for research purposes.

I am working with our local leadership to determine further action on the 5 patients and will reply to the ticket when I have more guidance. Thank you!

**Comment by Jackson-Barlocker** [comment]
@sxinger I have uploaded our newest pcornet_trial.sas7bdat file to the gpc-uu-upload S3 bucket, containing the 8 missing patients that you sent me via e-mail.

Please let me know if you're able to see the patients. Thank you!

**Comment by sxinger** [comment]
@Jackson-Barlocker thank you very much! 

@soliman455 @turabieh please help stage this new PCORNET_TRIAL table from UU.

**Comment by Jackson-Barlocker** [comment]
@sxinger, today we received approval for our data curation - which includes the 5 VCCC patients. May we resubmit this data to GROUSE and have all of our data restaged for use in DROC 203? Otherwise, even though the PCORNET_TRIAL will have these 5 patients, they will not be in our GROUSE data.

**Comment by sxinger** [comment]
@Jackson-Barlocker yes, please resubmit the trial table with all the recruits. Thanks a lot!

**Comment by Jackson-Barlocker** [comment]
@sxinger, I have begun our data reload into the gpc-uu-upload bucket with the command:
aws s3 cp . s3://gpc-uu-upload --recursive --exclude "lds_address_history.sas7bdat" --exclude "hash_token.sas7bdat" --profile GPC-UU-Data-Load

This will provide all of our curated data for use in the DROC 203 request, if we can get a refresh of our data on GROUSE. The previous PCORNET_TRIAL we sent over had all VCCC patients needed for the study but with our curated files, they will be in our DEMOGRAPHIC, ENCOUNTER, and other tables too.

The upload process should complete by our morning GPC call. I will notify if there are any issues. Thanks!

**Comment by sxinger** [comment]
@Jackson-Barlocker much appreciated! @turabieh please work with @soliman455  to get the data staged onto snowflake
